drivers/ethernet/eth_stm32_hal.c

```diff
@@ -32,18 +32,112 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #error DTCM for DMA buffer is activated but zephyr,dtcm is not present in dts
 #endif
 
+static HAL_StatusTypeDef ETH_RD_PHY_REG( ETH_HandleTypeDef* ps_heth,
+                                         uint32_t u32_PHYAddr,
+                                         uint32_t u32_PHYReg,
+                                         uint32_t* pu32_RegVal);
+
+#ifndef PHY_ADDR
+#define PHY_ADDR        ((uint32_t)CONFIG_ETH_STM32_HAL_PHY_ADDRESS)
+#endif
+
+#ifndef PHY_BSR
+#define PHY_BSR     ((uint16_t)0x0001U)
+#endif
+
+#ifndef PHY_LINKED_STATUS
+#define PHY_LINKED_STATUS       ((uint16_t)0x0004U)
+#endif
+
+#if defined(CONFIG_SOC_SERIES_STM32H7X)
+/**
+ * Definitions for ethernet driver of all others STM32 except STM32H7
+ */
+
+#define GET_TX_DESC_LIST_PTR(ps_heth)     (&(((ETH_HandleTypeDef*)ps_heth)->TxDescList))
+
+#define GET_FIRST_DMA_TX_DESC(ps_heth)                ((ETH_DMADescTypeDef*)(((ETH_HandleTypeDef*)ps_heth)->Init.TxDesc))
```

Comment:
Too long lines here and below. Could you go through the whole eth driver file and try to shorten them to 80 chars.

Reply:
> Too long lines here and below. Could you go through the whole eth driver file and try to shorten them to 80 chars.

done

Reply:
done